articles/concepts/advanced-matrices.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Advanced matrix concepts | Microsoft Docs
-description: Advanced matrix concepts
+title: Advanced matrix concepts
+description: Learn about eigenvectors, eigenvalues and exponentials, the fundamental tools used to describe and implement quantum algorithms. 
```

Comment:
```suggestion
description: Learn about eigenvectors, eigenvalues, and matrix exponentials, the fundamental tools used to describe and simulate quantum algorithms. 
```
By way of context, linear algebra is not used to implement quantum computing, but to describe and predict what a quantum computer will do in response to a sequence of instructions.